iscript/src/iscript/data/i_task_schema.json

```diff
@@ -99,6 +101,35 @@
             "entitlements-url": {
               "type": "string"
             },
+            "signing-config": {
+              "type": "array",
```

Comment:
Can you also point to an example of what this might look like in practice? It would help to understand how the schema fits together.

Reply:
It will be a variation of this:
https://github.com/hafta/codesign-tree/blob/master/examples/01-firefox/codesign-map.json